packages/ndla-ui/src/SearchTypeResult/SearchHeader.tsx

```diff
@@ -86,9 +88,16 @@ const SearchHeader = ({
     </SearchInputWrapper>
     <PhraseWrapper>
       {searchPhrase && (
-        <PhraseText>
-          {t('searchPage.resultType.showingSearchPhrase')} {searchPhrase}
-        </PhraseText>
+        <>
```

Comment:
Synes denne er litt vanskelig å lese. Kunne det kanskje vært en ide å flytte t-verdiene ut av render? Da vil man sitte igjen med to one-liners.